Server/Admin/routers/admin_auth.py

```
@@ -693,23 +707,53 @@ def admin_password_reset_confirm(data: AdminPasswordResetConfirm, request: Reque
-        raise HTTPException(status_code=404, detail="User not found.")
+        # Try to extract from Authorization header or cookie
+        from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+        from fastapi import Cookie
+        security = HTTPBearer()
+        
+        # Try to get token from header first
+        authorization: Optional[HTTPAuthorizationCredentials] = None
+        try:
+            authorization = security(request)
+        except Exception:
+            pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)